api/net/Areas/Services/Controllers/ContentController.cs

```diff
@@ -222,6 +222,8 @@ public async Task<IActionResult> UpdateAsync(ContentModel model, bool index = fa
     {
         var content = _contentService.UpdateAndSave((Content)model);
 
+        await _kafkaMessenger.SendMessageAsync(_kafkaHubOptions.HubTopic, new KafkaHubMessage(HubEvent.SendAll, new InvocationMessage("Content", new[] { new ContentMessageModel(content) })));
```

Comment:
Send it immediately after updating so that UI gets the message in the correct order